features/machine/machine.feature

```diff
@@ -141,4 +141,36 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
-    
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
+  Scenario: [MAO] Reconciling machine taints with nodes
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+
+    Given I store the number of machines in the :num_to_restore clipboard
+    And admin ensures node number is restored to "<%= cb.num_to_restore %>" after scenario
+
+    Given I clone a machineset named "machineset-24363"
+    When I store the last provisioned machine in the :machine clipboard
+    And evaluation of `machine(cb.machine).node_name` is stored in the :noderef_name clipboard
+
+    Given I saved following keys to list in :taintsid clipboard:
+      | {"spec":{"taints": [{"effect": "NoExecute","key": "role","value": "master"}]}}  | |
+      | {"spec":{"taints": [{"effect": "NoSchedule","key": "role","value": "master"}]}} | |
+
+    And I use the "openshift-machine-api" project
+    Then I repeat the following steps for each :id in cb.taintsid:
+    """
+    Given as admin I successfully merge patch resource "machine/<%= cb.machine %>" with:
+    | #{cb.id} |
+    Then the step should succeed
+    """
+    When I run the :describe admin command with:
+      | resource      | node                 |
+      | name          |<%= cb.noderef_name %>|
```

Comment:
Extra white spaces before the second `|`

Reply:
updated

Reply:
`|<%= cb.noderef_name %>|` -> `| <%= cb.noderef_name %> |`
@miyadav you can take advantage of some tooling to automate the table aligning, see http://vimcasts.org/episodes/aligning-text-with-tabular-vim/. This is to avoid reviewing the table format in your future PRs.